tasks/main-clevis.yml

```diff
@@ -56,4 +68,26 @@
     - __nbde_client_clevis_check_state is defined
     - __nbde_client_clevis_check_state is changed
 
+- name: Set-up network flushing
+  block:
+    - name: Deploy network flushing script
+      copy:
+        src: "{{ role_path }}/files/nbde_client-network-flush"
+        dest: /usr/bin/nbde_client-network-flush
+        mode: '0755'
+
+    - name: Deploy network flushing service
+      copy:
+        src: "{{ role_path }}/files/nbde_client-network-flush.service"
+        dest: /etc/systemd/system/nbde_client-network-flush.service
```

Comment:
I think using `mode: '0644'` will get rid of the warning

Reply:
Done.